EIPS/eip-x.md

```diff
@@ -0,0 +1,168 @@
+---
+eip: <to be assigned>
```

Comment:
Please use 1753.

Reply:
Fixed

Reply:
I think you forgot to push the changes?

Reply:
Ok, that's strange.  The pending commit must be on the other computer.   Thanks for following this up.

Reply:
Fixed